bylaws.md

```diff
@@ -455,6 +455,12 @@ cohort. Replacement directors shall be chosen in order of the number
 of votes received, with the longest terms of service being allocated
 to candidates according to the number of votes received.
 
+Persons elected as at-large directors are considered to be seated
+in order from most votes received to the least. If a person who would
+otherwise be elected withdraws or becomes ineligible before that person
+is seated as a director, then the person receiving the next highest
+number of votes is selected.
```

Comment:
Should we also add clarification about what should happen in the event of a tie?

Reply:
This was responded to in the meeting and the decision was that it is documented in the psf-election repo and does not need to be in the bylaws